### PR TITLE
server: tiny update and speed up test.

### PR DESCRIPTION
**Pull request by siddontang**
@ngaut  @disksing

### CODE REVIEW

**Comment by ngaut** [comment]
LGTM

File: server/leader.go [hunk 1]
```diff
@@ -67,7 +67,7 @@ func (s *Server) leaderLoop() {
 			log.Errorf("get leader err %v", err)
 			continue
 		} else if leader != nil {
-			log.Debugf("leader is %#v, watch it", leader)
+			log.Debugf("leader is %s, watch it", leader.String())
```
Comment (qiuyesuifeng) [inline server/leader.go]: Can we directly use `leader` instead of `leader.String()` as variable?

Comment (siddontang) [inline server/leader.go]: no, should use String to pretty print protocbuf struct.

**Comment by qiuyesuifeng** [comment]
LGTM

**Pull request merged**